examples/send_one.py

```diff
@@ -3,7 +3,10 @@
 
 
 def send_one():
-    bus = can.interface.Bus()
+    bus = can.interface.Bus(bustype='pcan', channel='PCAN_USBBUS1', bitrate=250000)
```

Comment:
Try not to commit your configuration - the `can.interface.Bus()` will load from configuration files and environment variables by default.

Reply:
But this comes from the master branch so i will not change something here.

Reply:
My bad!